app/shared/src/translations/Translate.js

```diff
@@ -1,7 +0,0 @@
-// @flow
-
-import { NativeModules } from 'react-native';
-
-export default function(key: string) {
```

Comment:
Not sure we should delete this entirely, What about cases like this? Or 3rd party components that expects strings.
```
return {
      headerLeft: renderHeaderLeft(),
      headerTitle: 'Hotels',
      headerRight: renderHeaderRight(),
    };
```

Reply:
My idea behind these changes is to change Flow types to enforce the translations. So for example in case of `headerTitle` the type is not going to be `string` but `TranslationType`. This way you cannot really make a mistake. Do you know about any 3rd party library where we need to use string and there is no other solution (so I can see it)? Because I think [we can use component in case of `headerTitle`](https://reactnavigation.org/docs/stack-navigator.html#headertitle).

Reply:
> Do you know about any 3rd party library where we need to use string

Nothing comes to mind. I am sure this is ok. If need be, we can always make it work